src/client.ts

```diff
@@ -2860,12 +2860,13 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
    *
    * @private
    * @param {string} path path of uploaded data
-   *
+   * @param {string} mode import mode insert|upsert
    * @return {APIResponse & CreateImportResponse} An ImportTask
    */
-  async _createImport(path: string) {
+  async _createImport(path: string, mode: string) {
```

Comment:
it might wiser to take second param as an option object so that we can add more flags in the future if needed

Reply:
since it's noted to be changed, not strict though